Examples/VaporExample/Resources/Views/index.leaf

```diff
@@ -55,8 +57,38 @@
     </select>
   </div>
   <div>
-    <button>Add</button>
+    <button id="add-btn">Add</button>
   </div>
 </form>
+
+<script type="text/javascript">
+  $(document).ready(() => {
+    $("\#add-btn").click(() => {
+      var name = $("\#name").val();
+      var color = $("\#color").val();
+      var food = $("\#favoriteFood").val();
+      // Serialize request data to extended JSON.
+      var data = BSON.EJSON.stringify({
+        'name': name,
+        'color': color,
+        'favoriteFood': food
```

Comment:
Would it be worth including like a "last updated date" or "birth date" or something here to demonstrate why the extended json is useful? these types are all strings so I think the output is pretty much the same with either encoder. If we do go that route, it might make sense to call it out specifically above that ext JSON is what ensures our dates get moved from JS frontend <-> Swift backend <-> MongoDB seamlessly.